data/overview.yaml

```diff
@@ -115,30 +115,30 @@ Groups:
     orthonormal basis of irreducible characters: ''
     examples for groups with a small cardinal: ''
 # 3.
-Rings:
-  ring: 'algebra/ring.html#ring'
+Ring Theory:
+  Ring: 'algebra/ring.html#ring'
   Unitary ring:
-  ring morphisms: 'algebra/ring.html#ring_hom'
-  subrings: 'ring_theory/subring.html#top'
+  Ring morphisms: 'algebra/ring.html#ring_hom'
+  Subrings: 'ring_theory/subring.html#top'
   The ring $\Z$ on the integers:
   Product of rings: 'algebra/pi_instances.html#pi.ring'
   Ideal of a commutative ring: 'algebra/module.html#ideal'
-  quotient rings: 'ring_theory/ideals.html#ideal.quotient'
-  prime ideals: 'ring_theory/ideals.html#ideal.is_prime'
-  maximal ideals: 'ring_theory/ideals.html#ideal.is_maximal'
+  Quotient rings: 'ring_theory/ideals.html#ideal.quotient'
+  Prime ideals: 'ring_theory/ideals.html#ideal.is_prime'
+  Maximal ideals: 'ring_theory/ideals.html#ideal.is_maximal'
   Chinese remainder theorem:
   Algebra:
     algebra over a commutative ring: 'ring_theory/algebra.html#algebra'
     associative algebra over a commutative ring:
-  Divisibility in commutative integral rings: ''
+  Divisibility in integral domains: ''
   Irreducible elements: 'algebra/associated.html#irreducible' 
   Invertible elements: 'algebra/invertible.html#invertible'
   Coprime elements: 'ring_theory/ideals.html#ideal.is_coprime'
   Unique factorisation domain (UFD): 'ring_theory/unique_factorization_domain.html'
   Greatest common divisor: 'algebra/gcd_domain.html#gcd_domain.gcd'
   Least common multiple: 'algebra/gcd_domain.html#gcd_domain.lcm'
   $A[X]$ is a UFD when $A$ is a UFD: ''
-  Principal ring: 'ring_theory/principal_ideal_domain.html#submodule.is_principal'
+  Principal ideal domain: 'ring_theory/principal_ideal_domain.html#submodule.is_principal'
```

Comment:
Note that this was previously discussed [here](https://github.com/leanprover-community/leanprover-community.github.io/pull/27#discussion_r429595741).